framework/include/materials/ADMaterial.h

```diff
@@ -17,7 +17,16 @@
   using ADMaterial<compute_stage>::_qp;                                                            \
   using ADMaterial<compute_stage>::_ad_grad_zero;                                                  \
   using ADMaterial<compute_stage>::_ad_zero;                                                       \
-  using ADMaterial<compute_stage>::coupledComponents
+  using ADMaterial<compute_stage>::_qrule;                                                         \
+  using ADMaterial<compute_stage>::_JxW;                                                           \
+  using ADMaterial<compute_stage>::_coord;                                                         \
+  using ADMaterial<compute_stage>::_assembly;                                                      \
+  using ADMaterial<compute_stage>::_mesh;                                                          \
+  using ADMaterial<compute_stage>::coupled;                                                        \
+  using ADMaterial<compute_stage>::coupledComponents;                                              \
+  using ADMaterial<compute_stage>::isParamValid;                                                   \
+  using ADMaterial<compute_stage>::getParam;                                                       \
```

Comment:
Ugh, we have `adGetParam`, but I think adding the `using` declarations makes for more intuitive code (especially when converting over non-AD classes. Plus we currently have two parallel mechanisms of removing `this->template ...` - the `using` stuff and the `ad...` macros). @lindsayad